uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/browser/IBrowser.java

```diff
@@ -34,6 +36,6 @@
      *
      * @param url the url to open
      */
-    void navigateTo(String url);
+    void navigateTo(String url) throws UiObjectNotFoundException;
```

Comment:
I don't think an exception is thrown here....why do we need the throws clause?

Reply:
yes, initially I used the .click() method that's why it is causing an exception.